operator/duties/dutystore/voluntary_exit.go

```diff
@@ -0,0 +1,44 @@
+package dutystore
+
+import (
+	"sync"
+
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+)
+
+type VoluntaryExitDuties struct {
+	mu sync.RWMutex
+	m  map[phase0.Slot]map[phase0.BLSPubKey]int
```

Comment:
we need to clean this map at some point, maybe remove the duties after they are executed? or that's too short?

Reply:
done, let's clean them after 1 epoch